### PR TITLE
use owner address to store vote bucket

### PR DESCRIPTION
**Pull request by dustinxie**

### CODE REVIEW

File: action/protocol/staking/vote_bucket.go [hunk 1]
```diff
@@ -22,9 +23,16 @@ import (
 )
 
 type (
-	// VoteBucket is an alias of proto definition
+	// VoteBucket represents a vote
 	VoteBucket struct {
-		iotextypes.Bucket
+		Candidate        hash.Hash160
+		Owner            hash.Hash160
```
Comment (CoderZhi) [inline action/protocol/staking/vote_bucket.go]: address.Address

File: action/protocol/staking/vote_bucket.go [hunk 2]
```diff
@@ -22,9 +23,16 @@ import (
...
+		StakedAmount     *big.Int
+		StakedDuration   uint32
```
Comment (CoderZhi) [inline action/protocol/staking/vote_bucket.go]: time.Duration

File: action/protocol/staking/vote_bucket.go [hunk 3]
```diff
@@ -108,7 +173,7 @@ func stakingGetTotalCount(sr protocol.StateReader) (uint64, error) {
 	return tc.count, err
 }
 
-func stakingGetBucket(sr protocol.StateReader, name CandName, index uint64) (*VoteBucket, error) {
+func stakingGetBucket(sr protocol.StateReader, name hash.Hash160, index uint64) (*VoteBucket, error) {
```
Comment (CoderZhi) [inline action/protocol/staking/vote_bucket.go]: address.Address and same for the rest of this PR

File: action/stake_create.go [hunk 4]
```diff
@@ -28,7 +30,7 @@ const (
 type CreateStake struct {
 	AbstractAction
 
-	candName  string
+	candName  hash.Hash160
 	amount    *big.Int
 	duration  uint32
```
Comment (CoderZhi) [inline action/stake_create.go]: time.Duration

File: action/stake_create.go [hunk 5]
```diff
@@ -28,7 +30,7 @@ const (
 type CreateStake struct {
 	AbstractAction
 
-	candName  string
+	candName  hash.Hash160
```
Comment (CoderZhi) [inline action/stake_create.go]: address.Address

File: action/stake_create.go [hunk 6]
```diff
@@ -68,7 +79,7 @@ func (cs *CreateStake) Amount() *big.Int { return cs.amount }
 func (cs *CreateStake) Payload() []byte { return cs.payload }
 
 // CandName returns the candidate name
-func (cs *CreateStake) CandName() string { return cs.candName }
+func (cs *CreateStake) CandName() hash.Hash160 { return cs.candName }
```
Comment (CoderZhi) [inline action/stake_create.go]: Candidate() address.Address

File: action/stake_create.go (hunk 4 above)
Comment (dustinxie) [reply action/stake_create.go]: fixed

File: action/protocol/staking/vote_bucket.go (hunk 1 above)
Comment (dustinxie) [reply action/protocol/staking/vote_bucket.go]: changed

File: action/protocol/staking/vote_bucket.go (hunk 2 above)
Comment (dustinxie) [reply action/protocol/staking/vote_bucket.go]: changed

File: action/protocol/staking/vote_bucket.go (hunk 3 above)
Comment (dustinxie) [reply action/protocol/staking/vote_bucket.go]: changed

File: action/stake_create.go (hunk 5 above)
Comment (dustinxie) [reply action/stake_create.go]: changed

File: action/stake_create.go (hunk 6 above)
Comment (dustinxie) [reply action/stake_create.go]: changed

**Pull request merged**